.gitmodules

```diff
@@ -31,3 +31,6 @@
 [submodule "lib/sdl/SDL2_image"]
 	path = lib/sdl/SDL2_image
 	url = https://github.com/libsdl-org/SDL_image.git
+[submodule "lib/usb/ohci_usbhostlib"]
+	path = lib/usb/ohci_usbhostlib
+	url = https://github.com/Ryzee119/ohci_usbhostlib.git
```

Comment:
On your master branch, could you rename your commit about original import of OpenNuvoton/N9H30_emWin_NonOS.git so it has a full URL + a commit hash for the latest version?

The commit hash is critical so we can later retrace where the fork began.

The URL change would be good because I'm concerned it will confuse users when the repository migrates elsewhere + it will actually make it easier to find it quickly.

Reply:
>Most of my feedback so far is overly critical, because it's about long-term effects of using this codebase / design that we'll inherit. Take it with a grain of salt.

I can appreciate this. Thanks for taking the time to review.

> I'll just use this top level comment to express my excitement about this changeset! Very good to see this done.

Very greatful to hear this. Thank you.